modules/ROOT/pages/persistence-gateway.adoc

```diff
@@ -0,0 +1,122 @@
+= Persistence Gateway
+
+To store state across applications, Anypoint Runtime Fabric provides the Persistence Gateway. Persistene Gateway enables applications deployed to a Mule runtime engine (Mule) to store and share data across batch processes, Mule components and applications.
```

Comment:
How about rephrasing this to:
Persistence Gateway feature of Anypoint Runtime Fabric enables applications deployed to a Mule runtime engine (Mule) to store/persist and share the state/data across batch processes, Mule components and applications.